pkg/pkg.bzl

```diff
@@ -292,11 +292,11 @@ pkg_tar_impl = rule(
             providers = [PackageVariablesInfo],
         ),
         "stamp": attr.int(
-            doc = """Enable file time stamping.  Possible values:<ul>
-<li>stamp = 1: Use the time of the build as the modification time of each file in the archive.</li>
-<li>stamp = 0: Use an "epoch" time for the modification time of each file. This gives good build result caching.</li>
-<li>stamp = -1: Control the chosen modification time using the --[no]stamp flag.</li>
-</ul>""",
+            doc = """Enable file time stamping.  Possible values:
+<li>stamp = 1: Use the time of the build as the modification time of each file in the archive.
+<li>stamp = 0: Use an "epoch" time for the modification time of each file. This gives good build result caching.
+<li>stamp = -1: Control the chosen modification time using the --[no]stamp flag.
```

Comment:
Could this just be markdown instead?

Reply:
Sadly, no. It can't.  https://github.com/bazelbuild/stardoc/issues/118